learn/contributing/contributing_docs.mdx

```diff
@@ -11,13 +11,26 @@ Our documentation aims to be:
 - **Thorough**: the documentation website should contain all information anyone needs to use Meilisearch
 - **Open source**: this is a resource by Meilisearch users, for Meilisearch users
 
+## Documentation repository and local development
+
+The Meilisearch documentation repository only stores the content of the docs website. Because the code that makes up the website lives in another repository, **it is not possible to run a local copy of the documentation**.
+
+### Handling images and other static assets
+
+When contributing content to the Meilisearch docs, store screenshots, images, GIFs, and videos in the relevant directory under `/assets`. 
+
+The build process does not currently support static assets with relative paths. When adding them to a document, make sure the asset URL points to the raw GitHub file address:
+
+```markdown
+\!\[Image description\]\(https://raw.githubusercontent.com/meilisearch/documentation/[branch_name]/assets/images/[guide_name]/diagram.png\)
+```
+
 ## How to contribute?
 
 ### Issues
 
 The maintainers of Meilisearch's documentation use [GitHub Issues](https://github.com/meilisearch/documentation/issues/new) to track tasks. Helpful issues include:
 
-- Bug reports detailing technical issues with this website
```

Comment:
Could we add something to the list about error spotting? Right now we're saying "just use issues for requests", but we also want people to use issues to notify us of things that are inaccurate, outdated, confusing etc.